uint8 id
uint16 position
uint16 voltage
int8 temperature
